src/radha/ReturnTypeArray/Assignment25/FindStringWithNumber.java

```diff
@@ -0,0 +1,34 @@
+/*
+Program 2: Write a method which will return all String from given array which contains atleast one number. Main method should print the output.
+Input : {"TechnoCredits", "HP1w", "Pd3R4K9", "Krishna", "Aashvi25"}
+Output : HP1w Pd3R4K9 Aashvi25
+ */
+package radha.ReturnTypeArray.Assignment25;
+
+public class FindStringWithNumber {
+
+	String[] findStringWithNumber(String[] str) {
+		String[] output = new String[str.length];
+		for (int index = 0; index < str.length; index++) {
+			for (int strIndex = 0; strIndex < str[index].length(); strIndex++) {
+				char letter = str[index].charAt(strIndex);
+				if (Character.isDigit(letter)) {
+					output[index] = str[index];
```

Comment:
for output indexing you should use counter variable, as jha pr if condition me nhi jayega idex pr vha null aayega in output array